docs/docs/user/tools/capella/migration/update.md

```diff
@@ -12,19 +12,20 @@
 
 <!-- prettier-ignore -->
 !!! info
-    If you want to update a Capella model saved in TeamForCapella, please follow the Capella specific instructions instead:
-    [Update a TeamForCapella based Capella model](../../tools/capella/teamforcapella/update.md)
+    If you want to update a Capella model saved in TeamForCapella, please follow the TeamForCapella specific instructions instead:
+    [Update a TeamForCapella based Capella model](../../../../admin/teamforcapella/migration/update.md).
+    An administrator is required to update the Capella version in TeamForCapella.
```

Comment:
If the administrator bit is only relevant to the process for TeamForCapella I'd delete it from this page (presumably it's in the linked page)

Reply:
This page is in the user documentation, so a user may expect to be able to follow the documentation from this point, but the referenced page is part of the administrator guide. 

A more likely workflow would be that the users reaches the reference, sends the link to an admin and asks to proceed. Therefore, it's important to mention that those steps can't be executed by "normal" users.